### ISSUE TITLE
Generate Exit Validator messages

### ISSUE THREAD

**Issue by rajkat1001** [ethereum/staking-deposit-cli#334]
Staking deposit cli should be modified to generate signed EVM (exit validator message json files) by taking in the mnemonics + mnemonic-password + validator-signing-passphrase.

Please make this tool as worthy of using as ethdo is said by many to be. But unfortunately many have used the launchpad staking-deposit tools with Windows support and are left out so disappointed now.

Windows version to go with the launchpad program highly needed. Syntax:

./staking-deposit-cli.exe generate-exit-validator mnemonic-phrase=xx,yy,zz.....aa --mnemonic-password=mpassword validator-password=valpassword

All other functions like current bls-to-execution-key-change etc.. to remain as they are.

If you want to see how such EVM is generated from mnemonics+mnemonic-password and the validator-password, please check the code implemented for same funciton in Ethdo.

**Comment by uwuforever** [comment]
Generating the actual message is not hard, but you need beacon node RPC access to broadcast it. The rational is, if you have a beacon node synced and set up, you could just use it to exit directly, and if you don't, you'll have to get one to use the signed message anyways. But free message publishing will soon be easily available, and I would expect the validator exit tools to follow shortly :)

As for the differences, in an exit message, you sign the epoch of your exit, and in a BLS-withdrawal message, you sign your old BLS public key and your new withdrawal address. It is incredibly similar.

**Comment by rajkat1001** [comment]
One does not need to know much technical details if one can use a signed message developed off deposit tool and directly publish on beaconcha.in or some similar interface.

In my case, the staking machine is Linux. The widows machine is just used offline to make the deposit, keystore, withdrawal and voluntary json exit messages.

**Comment by uwuforever** [comment]
> ...and directly publish on beaconcha.in or some similar interface.

Beaconcha.in only started doing that like a week ago, and only on the Zhejiang testnet at the moment. Before that you either ran your own beacon node server and used your person RPC endpoint/CLI client,

**Comment by rajkat1001** [comment]
Yes that's true. I ran my own beacon node used own endpoints and local cli interface. 

My hope is that the developers of staking-deposit-cli add support well in advance for generating VEMs without having to depend on the various concensus client interfaces or having to use them to broadcast the message in expectation that same can be submitted through beaconcha.in or some other public tool in near future. 

Obviously everyone is not technical enough and if some mechanism is soon provided, it will be easier to pass on the information to one's heirs or would be successors who want to manage these assets without all the pain of learning a whole lot of things. This is what the hardware wallets and other desktop wallets were designed for.. to make it easy to generate and publish signed messages. 

As long as it is understood that  VEM should be able to be generated by the deposit-cli and (w)could be implemented before mainnet comes alive, I am saying that it will be a well served cause for the community.
